pydicom/tests/test_numpy_pixel_data.py

```diff
@@ -245,6 +245,31 @@ def test_big_endian_PixelArray(self):
             "all {0} (mean == {1})".format(b.mean(), a.mean()))
 
 
+@pytest.mark.skipif(
+    not have_numpy_handler,
+    reason=numpy_missing_message)
+class numpy_LittleEndian_Tests(unittest.TestCase):
+    def setUp(self):
+        self.emri_small = dcmread(emri_name)
+        self.original_handlers = pydicom.config.image_handlers
```

Comment:
better move the read to the test that needs it

Reply:
In theory, this file would have multiple tests run on it in this class.  It doesn't really have that now, but that is the pattern followed by the other tests.  I'll update the new tests to match...